airflow/api_connexion/openapi/v1.yaml

```diff
@@ -369,6 +369,27 @@ paths:
         '404':
           $ref: '#/components/responses/NotFound'
 
+  /connections/{connection_id}/test:
```

Comment:
As Kaxil mentioned, it would be nice to have a test button for the edit form, so we'd need ` /connections/test` POST end point that creates the connection and tests it rather than loading one from the DB.

We possibly don't need this endpoint at all, as to me the use of an test button would be on the edit form, where we can just submit the full details payload.

Reply:
I tried that approach as well, but we are relying on the `get_hook` method from the connection class and then calling `test_connection` from the hook. But in some hooks the init looks for a valid connection and errors out if not found ex. https://github.com/apache/airflow/blob/master/airflow/providers/databricks/hooks/databricks.py#L120